deepcell_label/blueprints.py

```diff
@@ -64,6 +65,24 @@ def raw(token, channel, frame):
     return send_file(png, mimetype='image/png')
 
 
+@bp.route('/api/raw/<token>', methods=['POST'])
+def add_raw(token):
+    """Add new channel to the project."""
+    project = Project.get(token)
+    if not project:
+        return abort(404, description=f'project {token} not found')
+    # Load channel from first array in attached file
+    npz = np.load(request.files.get('file'))
+    channel = npz[npz.files[0]]
```

Comment:
Consider wrapping this in a try/except. What if the request doesn't have the `file`?

Reply:
I added blocks to catch `BadRequestKeyError` when the attached form does not have a file field and to catch `TypeError` for when `np.load` can't read the attached file.